src/main/java/com/authlete/common/util/Utils.java

```diff
@@ -80,7 +80,7 @@ public static String join(String[] strings, String delimiter)
         {
             sb.append(string);
 
-            if (useDelimiter);
```

Comment:
This is interesting... it's not just a style violation but looks like a bug was identified.